docs/services/qix-engine/access-control.md

```diff
@@ -605,3 +605,42 @@ user.region matches "us-[^-]+-(1|2)"
 
 This regular expression matches regions starting with `us-`, followed by one ore more characters that are anything but
 `-`, followed by `-1` or `-2`.
+
+## Adding existing apps
+
+By adding an existing app (qvf) to the engine running access control, you first have to import that app using engines api.
+This has to be done because the access control feature expects the app to follow a specific file structure.
+This structure is created when importing an app.
+
```

Comment:
Just some rephrasing, maybe sth like "If you want to add an existing app (qvf) to an engine running access control, you will have to import the app using the engine API. The access control feature expects the app to follow a specific file structure, which is created when an app is being imported."